### ISSUE TITLE
Update the readme to better describe the project

### ISSUE THREAD

**Issue by anubhav1206** [anubhav1206/blackjack-game#2]

**Comment by Ogochukwu-ugo** [comment]
Hi, @anubhav1206 I am new to programming, but I would love to work on this issue. Please, can you assign this task to me?

**Comment by anubhav1206** [comment]
> Hi, @anubhav1206 I am new to programming, but I would love to work on this issue. Please, can you assign this task to me?

Sure go ahead

**Comment by Khateebxtreme** [comment]
Hey, I was looking to make my initial contribution to get into open source. Can I and @Ogochukwu-ugo collaborate on this issue together to resolve it?

**Comment by anubhav1206** [comment]
Yes sure.

**Comment by Ogochukwu-ugo** [comment]
@Khateebxtreme Yes we can, please how can I reach out to you so you can see what I have done so far?
@anubhav1206 I tried pushing my update through the readme branch I created, and when I tried pushing to the branch, I got this error: 
remote: Permission to anubhav1206/blackjack-game.git denied to Ogochukwu-ugo.
fatal: unable to access 'https://github.com/anubhav1206/blackjack-game.git/': The requested URL returned error: 403

**Comment by Ogochukwu-ugo** [comment]
@anubhav1206 I have resolved the above error issue and also raised a PR.

**Comment by Khateebxtreme** [comment]
Hey, you can reach out to me on https://www.linkedin.com/in/khateebanwer/

**Comment by Khateebxtreme** [comment]
Hey, I saw the changes you did on your pull request and I came to a conclusion that it didn't require any further input from me. I think your pull request wasn't accepted because of the changes you made to the index.js file because apart from that it looks good.

**Comment by Khateebxtreme** [comment]
There are only minor typos or some grammatical errors that i can fix apart from that I would've done the same

**Comment by anubhav1206** [comment]
Thanks @Ogochukwu-ugo  I have merged your PR.

**Comment by anubhav1206** [comment]
Thanks @Khateebxtreme
